Purchases/Public/RCPurchases.m

```diff
@@ -96,7 +96,11 @@ + (BOOL)automaticAppleSearchAdsAttributionCollection {
 }
 
 + (void)setDebugLogsEnabled:(BOOL)enabled {
-    [self setLogLevel:RCLogLevelDebug];
+    if (enabled) {
+        [self setLogLevel:RCLogLevelDebug];
+    } else {
+        [self setLogLevel:RCLogLevelInfo];
+    }
```

Comment:
super nitpicky but I normally do ternary operators in cases like this. Of course feel free to ignore since it's my own preference :)

Reply:
I like it, updating